ccd-definition/ComplexTypes/CareSupervision/CaseManagementOrder/CaseManagementOrder.json

```diff
@@ -7,6 +7,15 @@
     "ElementLabel": "Order",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "UploadedCaseManagementOrder",
+    "ListElementCode": "lastUploadedOrder",
```

Comment:
Would wordVersionOfOrder or something like that be better, I know that it will be the last uploaded order but that name doesn't seem to fit right in my head

Reply:
As discussed on the Slack channel, I would like to leave the name as `lastUploadedOrder` which is clear that the order was uploaded by the user instead of a generated document